app/helpers/sessions_helper.rb

```diff
@@ -6,6 +6,15 @@ def log_in user
   # Returns the current logged-in user (if any).
   def current_user
     @current_user ||= User.find_by id: session[:user_id]
+    if (user_id = session[:user_id])
```

Comment:
```suggestion
    if user_id = session[:user_id]
```
hạn chế dùng ngoặc. chỗ nào bỏ dc thì bỏ nhé e